chpl/chpl-api/e2e/collections/certified-product-controller.postman_collection.json

```diff
@@ -1,54 +1,922 @@
+											"pm.test(\"/{chplPrefix}-{identifier} end point should return status 400 for requests without API Key\", function () {\r",
+											"    pm.response.to.have.status(400);\r",
+											"});"
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "GET",
+								"header": [
+									{
+										"key": "Content-Type",
+										"value": "application/json",
+										"type": "text"
+									}
+								],
+								"url": {
+									"raw": "{{url}}/rest/certified_products/CHP-028037",
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"rest",
+										"certified_products",
+										"CHP-028037"
+									]
+								},
+								"description": "GET /{chplPrefix}-{identifier} end point should return status 400 for requests without API Key"
+							},
+							"response": []
+						},
+						{
+							"name": "GET /{chplPrefix}-{identifier}/measures - Invalid API Key",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"/{chplPrefix}-{identifier}/measures end point should return status 401 for requests with Invalid API Key\", function () {\r",
+											"    pm.response.to.have.status(401);\r",
+											"});"
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "GET",
+								"header": [
+									{
+										"key": "API-Key",
+										"value": "1290y7y8h8ko90b8ecd0596c98ae9094",
+										"type": "text"
+									},
+									{
+										"key": "Content-Type",
+										"value": "application/json",
+										"type": "text"
+									}
+								],
+								"url": {
+									"raw": "{{url}}/rest/certified_products/CHP-028037/measures",
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"rest",
+										"certified_products",
+										"CHP-028037",
+										"measures"
+									]
+								},
+								"description": "GET /{chplPrefix}-{identifier}/measures end point should return status 401 for requests with Invalid API Key"
+							},
+							"response": []
+						},
+						{
+							"name": "GET /{chplPrefix}-{identifier}/measures - without API Key",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"/{chplPrefix}-{identifier}/measures end point should return status 400 for requests without API Key\", function () {\r",
+											"    pm.response.to.have.status(400);\r",
+											"});"
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "GET",
+								"header": [
+									{
+										"key": "Content-Type",
+										"value": "application/json",
+										"type": "text"
+									}
+								],
+								"url": {
+									"raw": "{{url}}/rest/certified_products/CHP-009321/measures",
```

Comment:
Why isn't this one a data-driven test?

Reply:
Added data driven tests for /chplPrefix-identifier end point